modules/update-best-practices.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-best-practices_{context}"]
+= Best practices for cluster updates
+
+{product-title} is designed to provide a robust update experience that allows clusters to update with minimal disruptions to workloads.
+Updates will not begin unless the cluster is determined to be in an upgradeable state at the time of the update request.
+
+This design ensures that some key conditions are met before initiating an update, but there are a number of actions you can take to increase your chances of a successful cluster update.
+
+[discrete]
+[id="recommended-versions_{context}"]
+=== Choose versions recommended by the OpenShift Update Service
```

Comment:
I can change it to be a second level heading, sure. I made it a third level for stylistic reasons but it's not super important to keep it that way. The reason they don't show up in the navigation on the right, however is because of the `[discrete]` elements, such as the one on line 14. 

I put those there because I thought it would overwhelm the sidebar navigation, but I guess if there are only 5 of these sections then that's okay for now!

Reply:
@skopacz1 I am sure you have updated the navigation level, but I can't see the change on preview page, could you help to confirm it?

Reply:
The automated preview page is not working correctly, sorry about that. I've manually uploaded the most recent preview with your feedback here: https://file.rdu.redhat.com/skopacz/OSDOCS-2427/updating/preparing_for_updates/updating-cluster-prepare.html#update-best-practices_updating-cluster-prepare

(you'll need to be on VPN to access this site)